sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpClient.cs

```diff
@@ -139,6 +142,7 @@ internal class AmqpClient : TransportClient
         /// <param name="identifier"></param>
         /// <param name="sessionId"></param>
         /// <param name="isSessionReceiver"></param>
+        /// <param name="transactionGroup"></param>
```

Comment:
nit:  These are all empty.   You probably want to either fill them out or remove them; they offer no value in this form.